### PR TITLE
remove unused and repeat code and remove TODO

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

move code due to they are only used by there   and  defined repeatly.

**Which issue(s) this PR fixes**:

ref:https://github.com/kubernetes/kubernetes/issues/86763
ref:https://github.com/kubernetes/kubernetes/issues/86052

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

File: test/e2e/e2e.go [hunk 1]
```diff
@@ -232,7 +240,7 @@ func setupSuite() {
 			framework.Failf("Error deleting orphaned namespaces: %v", err)
 		}
 		klog.Infof("Waiting for deletion of the following namespaces: %v", deleted)
-		if err := framework.WaitForNamespacesDeleted(c, deleted, framework.NamespaceCleanupTimeout); err != nil {
```
Comment (tanjunchen) [inline test/e2e/e2e.go]: it is only used here

File: test/e2e/framework/autoscaling/autoscaling_utils.go [hunk 2]
```diff
@@ -45,7 +45,6 @@ import (
 
 const (
 	dynamicConsumptionTimeInSeconds = 30
-	staticConsumptionTimeInSeconds  = 3600
```
Comment (tanjunchen) [inline test/e2e/framework/autoscaling/autoscaling_utils.go]: unused code

File: test/e2e/framework/deployment/fixtures.go [hunk 3]
```diff
@@ -33,7 +33,7 @@ import (
 
 // UpdateDeploymentWithRetries updates the specified deployment with retries.
 func UpdateDeploymentWithRetries(c clientset.Interface, namespace, name string, applyUpdate testutils.UpdateDeploymentFunc) (*appsv1.Deployment, error) {
-	return testutils.UpdateDeploymentWithRetries(c, namespace, name, applyUpdate, framework.Logf, poll, pollShortTimeout)
+	return testutils.UpdateDeploymentWithRetries(c, namespace, name, applyUpdate, framework.Logf, framework.Poll, framework.PollShortTimeout)
```
Comment (neolit123) [inline test/e2e/framework/deployment/fixtures.go]: i don't think i like this coupling between framework sub-packages and the root framework WRT timeouts.
keeping timeouts specific to a package makes more sense even if there are duplicates.

File: test/e2e/storage/pvc_protection.go [hunk 4]
```diff
@@ -36,6 +36,11 @@ import (
 	"k8s.io/kubernetes/test/e2e/storage/utils"
 )
 
+const (
+	// claimDeletingTimeout is How long claims have to become deleted.
+	claimDeletingTimeout = 3 * time.Minute
```
Comment (neolit123) [inline test/e2e/storage/pvc_protection.go]: this seems fine.

File: test/e2e/framework/util.go [hunk 5]
```diff
@@ -163,17 +153,13 @@ const (
 	// Minimal number of nodes for the cluster to be considered large.
 	largeClusterThreshold = 100
 
-	// TODO(justinsb): Avoid hardcoding this.
```
Comment (neolit123) [inline test/e2e/framework/util.go]: try separating such changes in separate PRs or separate commits.

File: test/e2e/framework/deployment/fixtures.go (hunk 3 above)
Comment (tanjunchen) [reply test/e2e/framework/deployment/fixtures.go]: sure

File: test/e2e/framework/util.go (hunk 5 above)
Comment (tanjunchen) [reply test/e2e/framework/util.go]: ok

File: test/e2e/framework/util.go [hunk 6]
```diff
@@ -171,9 +161,6 @@ const (
 )
 
 var (
-	// ClaimProvisionTimeout is how long claims have to become dynamically provisioned.
-	ClaimProvisionTimeout = 5 * time.Minute
```
Comment (tanjunchen) [inline test/e2e/framework/util.go]: move it to const not var

**Comment by oomichi** [comment]
Nice work, thanks for doing this.

/lgtm

**Review by SataQiu — COMMENTED** [review]
This seems ok! Thanks @tanjunchen 
/lgtm
/approve
